docs/config-file-ort-yml.md

```diff
@@ -6,7 +6,8 @@ increased degree of automation and local configurations should only be done if t
 
 * [excludes](#excludes) - Mark [files, directories](#excluding-paths) or [package manager scopes](#excluding-scopes) as
   not included in released artifacts.
-* [license finding curations](#curations) - Overwrite scan results to correct identified licenses.
+* [curations](#curations) - Curations correct invalid or missing package metadata and set the concluded license for
```

Comment:
The structure is changed now from

```
curations:
  license_findings:
```
to
```
curations:
  license_findings:
  packages:
```
The description now only describes package curations. I believe it should describe both.
Maybe with "Overwrite package meta-data, set a concluded license or correct license findings." or maybe separate sections for `package curations` and `license finding curations` would be more straight forward?